content/docs/06-troubleshooting.md

```diff
@@ -1,37 +1,35 @@
 # Troubleshooting
 
-If you're ever stuck with something, feel free to check if your issue is here. If your issue isn't here, or it's still not working, then join our [support server](https://promise.solutions/discord) and we can help you out.
+If you're experiencing issues with Countr, don't worry! We're here to help. Check out the common issues below to see if your problem is listed. If it's not, or if you're still having trouble, join our [support server](https://promise.solutions/discord) and we'll assist you further.
 
 
 ## "Remove everyone from a role" action is not working {#role-member-caching}
 
-To save up on memory, we don't cache everyone's user information, including their roles. We've tried to work around this issue by fetching everyone in the server that have a score in the counting channel, and then filter out who has the role and who doesn't, but this isn't always reliable. There's currently no way of fetching everyone from a role or set a role's members directly through the Discord API.
+If you're having trouble with the "Remove everyone from a role" action, it may be due to a memory-saving feature we use. We don't cache everyone's user information, including their roles. We've tried to work around this issue by fetching everyone in the server that have a score in the counting channel, and then filter out who has the role and who doesn't, but this isn't always reliable. Unfortunately, there's currently no way of fetching everyone from a role or setting a role's members directly through the Discord API.
```

Comment:
This could be rewritten to say something about Countr not tracking members, and Discord managing keeping the most accurate count very hard. Very large servers should use the "Give unique role" flow action instead.

Reply:
This flow action is deprecated anyways so idk why I didn't remove it sooner